README.md

```diff
@@ -2,13 +2,19 @@
 
 ## How to use
 
+To install:
+
+```
+npm i @fundingoptions/eslint-plugin-funding-options --save-dev
+```
+
 In .eslintrc.js
 
 ```js
 module.exports = {
-  plugins: ["eslint-plugin-funding-options"],
+  plugins: ["@fundingoptions/eslint-plugin-funding-options"],
```

Comment:
wouldn't it be so cool if it was `@fundingoptions/eslint-plugin`

Reply:
i'd prefer this as well, because we already have the `@funding-options` namespace. But appreciate it's a harder one to change